Sources/PointFree/TODO.swift

```diff
@@ -220,3 +220,37 @@ public func onclick<T>(javascript: StaticString) -> Attribute<T> {
 public func data<T>(_ name: StaticString, _ value: String) -> Attribute<T> {
   return .init("data-\(name)", value)
 }
+
+extension PartialIso {
+  /// Backwards composes two partial isomorphisms.
+  public static func <<< <C> (lhs: PartialIso<B, C>, rhs: PartialIso<A, B>) -> PartialIso<A, C> {
+    return .init(
+      apply: rhs.apply >-> lhs.apply,
+      unapply: lhs.unapply >-> rhs.unapply
+    )
+  }
+}
+
+extension UUID: RawRepresentable {
+  public var rawValue: String {
+    return self.uuidString
+  }
+
+  public init?(rawValue: String) {
+    guard let uuid = UUID(uuidString: rawValue) else { return nil }
+    self = uuid
+  }
+}
+
+extension PartialIso where A == B.RawValue, B: RawRepresentable {
+  public static var _rawRepresentable: PartialIso {
+    return .init(
+      apply: B.init(rawValue:),
+      unapply: ^\.rawValue
+    )
+  }
+}
+
+public func mapExcept<E, F, A, B>(_ f: @escaping (Either<E, A>) -> Either<F, B>) -> (EitherIO<E, A>) -> EitherIO<F, B> {
```

Comment:
at one point I made use of this, but doesnt seem like I have it anymore. tho im sure it could be useful